src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -361,31 +361,41 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 
 	private addDecoration(range: NotebookRange): void {
 		if (range && this.output && this.output.nativeElement) {
+			let markOutput = new Mark(this.output.nativeElement); // to highlight all occurances in the element.
```

Comment:
Maybe something like `markAllOccurances` instead? I'm not sure what output has to do with this - it definitely confused me since I thought it was referring to actual cell output. 

And then the mark below could be `markSelectedText`